src/javascripts/views/loginView.js

```diff
@@ -0,0 +1,62 @@
+import { MESSAGE, ERROR_MESSAGE } from "../constants/message";
+import { validate } from "../helpers/validationForm";
+
+class LoginView{
+    
+    constructor(){
+        this.loginForm = document.querySelector(".login-form");
+        this.buttonElement = document.querySelector(".login-form-button");
+    }
+
+    //----- EVENT LISTENER -----//
+    addEventLogin = (isValidAccount) => {
+        this.buttonElement?.addEventListener("click", async(e)=>{
+            e.preventDefault();
+
+            const accountInput = {
+                email: this.loginForm.email.value,
+                password: this.loginForm.password.value,
+            }
+
+            const inputs = validate(accountInput);
+            const isValidFields = this.isValidation(inputs);
```

Comment:
isValidation do 2 things into one function
1. Get input is error or not
2. Show error
=> Pls split it